contrib/verifysfbinaries/verify.sh

```diff
@@ -18,12 +18,12 @@ WORKINGDIR="/tmp/bitcoin"
 TMPFILE="hashes.tmp"
 
 #this URL is used if a version number is not specified as an argument to the script
-SIGNATUREFILE="https://bitcoin.org/bin/0.9.2.1/SHA256SUMS.asc"
+SIGNATUREFILE="https://bitcoinclassic.com/bin/0.9.2.1/SHA256SUMS.asc"
 
 SIGNATUREFILENAME="SHA256SUMS.asc"
 RCSUBDIR="test/"
-BASEDIR="https://bitcoin.org/bin/"
-VERSIONPREFIX="bitcoin-"
+BASEDIR="https://bitcoinclassic.com/bin/"
+VERSIONPREFIX="bitcoin-classic-"
```

Comment:
Apart from urls, it looks like "bitcoin-classic" has more instances within the source, rather than "bitcoinclassic".